### DISCUSSION TITLE
Embedded resources

### DISCUSSION THREAD

**Discussion by guija** [i-net-software/JWebAssembly#69] [General]
Hi, in your milestones section you mention that embedding resources is part of milestone 2. Is this already done?

My scenario looks like this:
- We have some java code base that we want to transpile to WASM.
- When code is executed it needs some master data accessible that needs to be loaded before execution.

Is this already somehow possible, e.g. either to embed the resource file in a static way or to download it from somewhere? I don't find anything related in the documentation.

Thanks.

**Comment by Horcrux7** [comment]
No, resource files was not startet yet. A possible workaround can be to inject it as byte array from JavaScript.
